### DISCUSSION TITLE
공지사항 타이틀 중복 출력 오류 제보합니다

### DISCUSSION THREAD

**Discussion by laterre39** [pronist/hELLO#133] [Ideas]
언제나 스킨 감사하게 사용하고있습니다 
개발자님 감사합니다.

오류 찾아서 재보하게 되었습니다

https://nworlds.tistory.com/notice

공지사항 타이틀이 중복으로 글 문단 위에 출력 되는 문제가 있습니다
확인 좀 부탁 드려도 될까요? 저만 그런데 아니라 다른 분들도 그러는 듯 해서
제보 올려봅니다.

**Comment by pronist** [comment]
@INewWorldI 안녕하세요.

이 문제는 점검해보아야 할 문제인 듯 싶습니다. 
확인해보니 이 문제는 버그로 확인되었습니다. 공지를 두 개 이상 등록하면 제목이 복사되는군요.

버그 신고 감사드립니다.
